src/main/java/org/folio/rest/impl/LoginAPI.java

```diff
@@ -103,6 +106,34 @@ public class LoginAPI implements Authn {
   public static final String MESSAGE_LOG_CONFIGURATION_IS_DISABLED = "Logging settings are disabled";
   public static final String MESSAGE_LOG_EVENT_IS_DISABLED = "For event logging `%s` is disabled";
   private static final String ERROR_EVENT_CONFIG_NOT_FOUND = "Event Config with `%s`: `%s` was not found in the db";
+  private static final String TOKEN_SIGN_ENDPOINT = "/token/sign";
+  private static final String TOKEN_SIGN_ENDPOINT_LEGACY = "/token";
+  private static final String TOKEN_REFRESH_ENDPOINT = "/token/refresh";
+
+  /**
+   * A time in the past which can be used in the Expires cookie attribute.
+   * Setting the cookie to a time in the past will cause the UA to delete it.
+   * See https://datatracker.ietf.org/doc/html/rfc6265#section-3.1.
```

Comment:
The URL should not have the trailing dot, otherwise the fragment identifier doesn't work. Consider using `<a href=""></a>` to make it clickable.

Reply:
Converted it to a hyperlink.